### ISSUE TITLE
Not able to reproduce the SNLI scores

### ISSUE THREAD

**Issue by atul04** [jihunchoi/unsupervised-treelstm#13]
Can I know the parameters settings. Please

**Comment by ayushbits** [comment]
Hi @jihunchoi ,

I am also not able to reproduce experiments on 300D embeddings **(85.6% Accuracy).**
I am getting around 82% with 300D and the hyperparams as mentioned in the paper.

Could you share  parameter file to reproduce SNLI experiments ?

Thanks,

**Comment by jihunchoi** [comment]
It is already uploaded in the following: https://github.com/jihunchoi/unsupervised-treelstm/tree/master/pretrained.
For loading the provided vocab and parameter files, I think https://github.com/jihunchoi/unsupervised-treelstm/blob/master/extract_trees.ipynb would be helpful.
Thanks!
